src/Raven.Client/Documents/Session/IAdvancedDocumentSessionOperations.cs

```diff
@@ -33,6 +33,8 @@ public interface IAdvancedDocumentSessionOperations
         RequestExecutor RequestExecutor { get; }
         JsonOperationContext Context { get; }
 
+        SessionInfo SessionInfo { get; }
```

Comment:
Do we need to expose this?

Reply:
Yes, this is commonly needed when we want to properly use `ExecuteAsync` for custom commands.